CardCollection/searchhandler.py

```diff
@@ -0,0 +1,22 @@
+from backend import Backend
+
+class SearchHandler:
+    """
+    A class that handles search operations by constructing queries and retrieving results.
+
+    Methods:
+        handle_search(search_parameters): Constructs a query string using the Backend class, construct_query function.
+    """
+    def handle_search(self, search_parameters: list[tuple[str, str, str]]) -> list[Card]:
+        """
+        Constructs a query string using the Backend class, construct_query function.
```

Comment:
Use this for the function description:

A function that takes a list of tuples representing search parameters, constructs a query string using the Backend class, and retrieves a list of Card objects from the API.